src/pages/_app.tsx

```diff
@@ -1,7 +1,10 @@
 import type { AppProps } from 'next/app'
 import { appWithTranslation } from 'next-i18next'
+import { SessionProvider } from 'next-auth/react';
 
 const MyApp = ({ Component, pageProps }: AppProps) => (
-  <Component {...pageProps} />
+  <SessionProvider>
+    <Component {...pageProps} />
+  </SessionProvider>
```

Comment:
:x
you PR should not have anything related to next-auth

Reply:
ok but in this situation i will have to delete some commits I made elated to login button components and the _app